README.md

```diff
@@ -413,6 +413,7 @@ Additional lists you might find useful:
 
 - [CakePHP Codeception module](https://github.com/cakephp/codeception) - The official CakePHP integration with [Codeception](https://codeception.com).
 - [CakePHP CodeSniffer rules](https://github.com/cakephp/cakephp-codesniffer) - The official CakePHP CS rules.
+- :strawberry: [CakePHP Fixture Factories plugin](https://github.com/pakacuda/cakephp-fixture-factories) - Create your fixtures dynamically on a test basis, accelerate the writing and maintainance of your tests.
```

Comment:
That is not the name, we need the 100% match here, I posted it for the reason to clarify. You used a different name now again after fixing the initially wrong one.